production/scrapers/indiana.R

```diff
@@ -1,16 +1,16 @@
 source("./R/generic_scraper.R")
 source("./R/utilities.R")
 
-indiana_date_check <- function(x, date = Sys.Date()){
-    base_html <- xml2::read_html(x)
+indiana_date_check <- function(url, date = Sys.Date()){
+    base_html <- xml2::read_html(url)
     
     base_html %>%
         rvest::html_nodes("p") %>%
         rvest::html_text() %>% 
-        {.[str_detect(., "(?i)last updated")]} %>% 
+        {.[str_detect(., "(?i)current as of")]} %>% 
         str_split("updated") %>% 
```

Comment:
I think we can remove this line and the following one since the text doesn't contain the word "updated" (at least right now). The date check function still works, with or without the string split, though!

Reply:
You're very correct, changed in latest commit!